app/javascript/packs/dashboard.js

```diff
@@ -2,10 +2,12 @@ import { widget as dashboardWidget } from 'Dashboard/index'
 import { initialize as toggleWidget } from 'Dashboard/toggle'
 import { render as renderChartWidget } from 'Dashboard/chart'
 import { ApiFilterWrapper } from 'Dashboard/components/ApiFilter'
+import { initialize as tabsWidget } from 'Dashboard/tabs-widget'
 
 document.addEventListener('DOMContentLoaded', () => {
   window.dashboardWidget = dashboardWidget
   window.toggleWidget = toggleWidget
   window.renderChartWidget = renderChartWidget
   window.ApiFilter = ApiFilterWrapper
+  window.tabsWidget = tabsWidget
```

Comment:
A good time to stop polluting the window and move the init fn here.

Reply:
Moved to its own pack, `api-as-product` that is to be merged into `dashboard` once the rolling update is removed.